src/Raven.Server/Rachis/Commands/Leader.RachisMergedCommand.cs

```diff
@@ -115,7 +115,14 @@ private void InsertCommandToLeaderLog(ClusterOperationContext context)
 
             private void AfterCommit(LowLevelTransaction tx)
             {
-                _leader._newEntry.Set();
+                try
+                {
+                    _leader._newEntry.Set();
+                }
+                catch (ObjectDisposedException ex)
```

Comment:
fix the warning

Reply:
can we move this forward please? Our production is having issues due to this one.